Code/Framework/AzCore/Tests/Platform/Windows/Tests/IO/Streamer/StorageDriveTests_Windows.cpp

```diff
@@ -605,7 +599,7 @@ namespace AZ::IO
             EXPECT_EQ(request.GetStatus(), AZ::IO::IStreamerTypes::RequestStatus::Completed);
             auto& readRequest = AZStd::get<AZ::IO::Requests::ReadData>(request.GetCommand());
             EXPECT_EQ(readRequest.m_size, fileSize);
-            EXPECT_STREQ(readRequest.m_path.GetAbsolutePath(), m_dummyFilepath.c_str());
+            EXPECT_STREQ(readRequest.m_path.GetAbsolutePath().Native().data(), m_dummyFilepath.c_str());
```

Comment:
We have Googletest Pretty Printers for the AZ Path classes(as well as the AZStd string classes): https://github.com/o3de/o3de/blob/development/Code/Framework/AzTest/AzTest/Printers.h#L35-L38

You can just directly use EXPECT_EQ here and if there is an error that occurs, it will output the expect and actual values as [strings](https://github.com/o3de/o3de/blob/development/Code/Framework/AzTest/AzTest/Printers.cpp#L16-L29)